### ISSUE TITLE
Validator chokes on unzipping when there's a lot of images

### ISSUE THREAD

**Issue by josteinaj** [nlbdev/nordic-epub3-dtbook-migrator#302]
Excerpt from detailed log with trace level output enabled:
```
2015-02-26 11:24:01,040 [INFO ] org.daisy.pipeline.job.Job - Changing job status to: RUNNING
2015-02-26 11:24:01,046 [INFO ] o.d.p.p.impl.job.PersistentJob - Changing Status:RUNNING
2015-02-26 11:24:01,046 [DEBUG] o.d.p.p.impl.job.PersistentJob - Updating object
2015-02-26 11:24:01,047 [DEBUG] o.d.p.p.i.job.PersistentJobContext - load script XProcScript[name=Nordic EPUB3 Validator]

...

2015-02-26 11:25:13,255 [TRACE] com.xmlcalabash.runtime.XPipeline - bundle://45.0:1/xml/xproc/mediatype.xpl:3:77:Pipeline output copy from [pipe #1363] ([document-sequence #446 (1 docs)]) to [pipe #678] ([document-sequence #399 (1 docs)])
2015-02-26 11:25:13,255 [TRACE] com.xmlcalabash.runtime.XForEach - Running p:for-each !1.17
2015-02-26 11:25:13,256 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 read 'bundle://70.0:1/xml/xproc/unzip-fileset.xpl' from [pipe #679] ([document-sequence #399 (1 docs)])
2015-02-26 11:25:13,268 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Input copy from xselect [document-sequence #400 (599 docs)]
2015-02-26 11:25:13,268 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:<d:file xmlns:d="http://www.daisy.org/ns/pipeline/data"
        xmlns:c="http://www.w3.org/ns/xproc-step"
        media-type="application/octet-stream"
        href="mimetype"/>

2015-02-26 11:25:13,268 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Input copy from xselect [document-sequence #400 (599 docs)]
2015-02-26 11:25:13,268 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:<d:file xmlns:d="http://www.daisy.org/ns/pipeline/data"
        xmlns:c="http://www.w3.org/ns/xproc-step"
        media-type="application/xhtml+xml"
        href="EPUB/324123-001-cover.xhtml"/>

...

2015-02-26 11:25:13,340 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Input copy from xselect [document-sequence #400 (599 docs)]
2015-02-26 11:25:13,340 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:<d:file xmlns:d="http://www.daisy.org/ns/pipeline/data"
        xmlns:c="http://www.w3.org/ns/xproc-step"
        media-type="image/jpeg"
        href="EPUB/images/324123-182-001.jpg"/>

2015-02-26 11:25:13,340 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Input copy from xselect [document-sequence #400 (599 docs)]
2015-02-26 11:25:13,340 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:<d:file xmlns:d="http://www.daisy.org/ns/pipeline/data"
        xmlns:c="http://www.w3.org/ns/xproc-step"
        media-type="image/jpeg"
        href="EPUB/images/324123-183-001.jpg"/>

2015-02-26 11:25:13,340 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Input copy from xselect [document-sequence #400 (599 docs)]
2015-02-26 11:25:13,340 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:<d:file xmlns:d="http://www.daisy.org/ns/pipeline/data"
        xmlns:c="http://www.w3.org/ns/xproc-step"
        media-type="image/jpeg"
        href="EPUB/images/cover.jpg"/>

2015-02-26 11:25:13,341 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Input copy from xselect [document-sequence #400 (599 docs)]
2015-02-26 11:25:13,341 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:<d:file xmlns:d="http://www.daisy.org/ns/pipeline/data"
        xmlns:c="http://www.w3.org/ns/xproc-step"
        media-type="application/octet-stream"
        href="EPUB/images/Thumbs.db"/>

2015-02-26 11:25:13,341 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Input copy from xselect [document-sequence #400 (599 docs)]
2015-02-26 11:25:13,341 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:<d:file xmlns:d="http://www.daisy.org/ns/pipeline/data"
        xmlns:c="http://www.w3.org/ns/xproc-step"
        media-type="application/x-dtbncx+xml"
        href="EPUB/nav.ncx"/>

2015-02-26 11:25:13,341 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Input copy from xselect [document-sequence #400 (599 docs)]
2015-02-26 11:25:13,341 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:<d:file xmlns:d="http://www.daisy.org/ns/pipeline/data"
        xmlns:c="http://www.w3.org/ns/xproc-step"
        media-type="application/xhtml+xml"
        href="EPUB/nav.xhtml"/>

2015-02-26 11:25:13,341 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Input copy from xselect [document-sequence #400 (599 docs)]
2015-02-26 11:25:13,341 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:<d:file xmlns:d="http://www.daisy.org/ns/pipeline/data"
        xmlns:c="http://www.w3.org/ns/xproc-step"
        media-type="application/oebps-package+xml"
        href="EPUB/package.opf"/>

2015-02-26 11:25:13,341 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Input copy from xselect [document-sequence #400 (599 docs)]
2015-02-26 11:25:13,341 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:<d:file xmlns:d="http://www.daisy.org/ns/pipeline/data"
        xmlns:c="http://www.w3.org/ns/xproc-step"
        media-type="application/xml"
        href="META-INF/container.xml"/>

2015-02-26 11:25:13,341 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Copy to current
2015-02-26 11:25:13,341 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #6160] ([document-sequence #2844 (1 docs)])
2015-02-26 11:25:13,342 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #6161] ([document-sequence #2844 (1 docs)])
2015-02-26 11:25:13,342 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:42:19:!1.17.3 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #6162] ([document-sequence #2844 (1 docs)])
2015-02-26 11:25:13,342 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:42:19:!1.17.3 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #6163] ([document-sequence #2844 (1 docs)])
2015-02-26 11:25:13,342 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:42:19:!1.17.3 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #6164] ([document-sequence #2844 (1 docs)])
2015-02-26 11:25:13,342 [DEBUG] com.xmlcalabash.extensions.Unzip - Running {http://www.daisy.org/ns/pipeline/xproc}unzip !1.17.3
2015-02-26 11:25:13,344 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:47:63:!1.17.4 read 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!mimetype' from [pipe #6165] ([document-sequence #401 (1 docs)])
2015-02-26 11:25:13,344 [DEBUG] com.xmlcalabash.library.AddAttribute - Running add-attribute !1.17.4
2015-02-26 11:25:13,344 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:47:63:!1.17.4 read 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!mimetype' from [pipe #681] ([document-sequence #401 (1 docs)])
2015-02-26 11:25:13,344 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 read 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!mimetype' from [pipe #683] ([document-sequence #402 (1 docs)])
2015-02-26 11:25:13,344 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 wrote 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!mimetype' to [pipe #685] ([document-sequence #404 (0 docs)])
2015-02-26 11:25:13,345 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Output copy from [pipe #683] ([document-sequence #402 (1 docs)]) to [pipe #685] ([document-sequence #404 (1 docs)])
2015-02-26 11:25:13,345 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Copy to current
2015-02-26 11:25:13,345 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #6166] ([document-sequence #2844 (1 docs)])
2015-02-26 11:25:13,345 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #6167] ([document-sequence #2844 (1 docs)])
2015-02-26 11:25:13,345 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:42:19:!1.17.3 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #6168] ([document-sequence #2844 (1 docs)])
2015-02-26 11:25:13,345 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:42:19:!1.17.3 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #6169] ([document-sequence #2844 (1 docs)])
2015-02-26 11:25:13,345 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:42:19:!1.17.3 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #6170] ([document-sequence #2844 (1 docs)])
2015-02-26 11:25:13,345 [DEBUG] com.xmlcalabash.extensions.Unzip - Running {http://www.daisy.org/ns/pipeline/xproc}unzip !1.17.3
2015-02-26 11:25:13,346 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:47:63:!1.17.4 read 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!EPUB/324123-001-cover.xhtml' from [pipe #6171] ([document-sequence #401 (1 docs)])
2015-02-26 11:25:13,346 [DEBUG] com.xmlcalabash.library.AddAttribute - Running add-attribute !1.17.4
2015-02-26 11:25:13,346 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:47:63:!1.17.4 read 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!EPUB/324123-001-cover.xhtml' from [pipe #681] ([document-sequence #401 (1 docs)])
2015-02-26 11:25:13,346 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 read 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!EPUB/324123-001-cover.xhtml' from [pipe #683] ([document-sequence #402 (1 docs)])
2015-02-26 11:25:13,346 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 wrote 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!EPUB/324123-001-cover.xhtml' to [pipe #685] ([document-sequence #404 (1 docs)])
2015-02-26 11:25:13,346 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Output copy from [pipe #683] ([document-sequence #402 (1 docs)]) to [pipe #685] ([document-sequence #404 (2 docs)])

...

2015-02-26 11:27:06,702 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Copy to current
2015-02-26 11:27:06,702 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #9250] ([document-sequence #2844 (1 docs)])
2015-02-26 11:27:06,702 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #9251] ([document-sequence #2844 (1 docs)])
2015-02-26 11:27:06,703 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:42:19:!1.17.3 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #9252] ([document-sequence #2844 (1 docs)])
2015-02-26 11:27:06,703 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:42:19:!1.17.3 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #9253] ([document-sequence #2844 (1 docs)])
2015-02-26 11:27:06,703 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:42:19:!1.17.3 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #9254] ([document-sequence #2844 (1 docs)])
2015-02-26 11:27:06,703 [DEBUG] com.xmlcalabash.extensions.Unzip - Running {http://www.daisy.org/ns/pipeline/xproc}unzip !1.17.3
2015-02-26 11:27:07,364 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:47:63:!1.17.4 read 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!EPUB/images/324123-155-006.jpg' from [pipe #9255] ([document-sequence #401 (1 docs)])
2015-02-26 11:27:07,364 [DEBUG] com.xmlcalabash.library.AddAttribute - Running add-attribute !1.17.4
2015-02-26 11:27:07,364 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:47:63:!1.17.4 read 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!EPUB/images/324123-155-006.jpg' from [pipe #681] ([document-sequence #401 (1 docs)])
2015-02-26 11:27:07,634 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 read 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!EPUB/images/324123-155-006.jpg' from [pipe #683] ([document-sequence #402 (1 docs)])
2015-02-26 11:27:07,634 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 wrote 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!EPUB/images/324123-155-006.jpg' to [pipe #685] ([document-sequence #404 (515 docs)])
2015-02-26 11:27:07,634 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Output copy from [pipe #683] ([document-sequence #402 (1 docs)]) to [pipe #685] ([document-sequence #404 (516 docs)])
2015-02-26 11:27:07,634 [TRACE] com.xmlcalabash.runtime.XForEach - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:Copy to current
2015-02-26 11:27:07,634 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #9256] ([document-sequence #2844 (1 docs)])
2015-02-26 11:27:07,635 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:39:17:!1.17 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #9257] ([document-sequence #2844 (1 docs)])
2015-02-26 11:27:07,635 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:42:19:!1.17.3 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #9258] ([document-sequence #2844 (1 docs)])
2015-02-26 11:27:07,635 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:42:19:!1.17.3 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #9259] ([document-sequence #2844 (1 docs)])
2015-02-26 11:27:07,635 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:42:19:!1.17.3 read 'file:/home/jostein/Skrivebord/daisy-pipeline-1.9/data/jobs/e142018d-112e-45cd-8106-cdfa6797bc61/output/temp-dir/validate/' from [pipe #9260] ([document-sequence #2844 (1 docs)])
2015-02-26 11:27:07,635 [DEBUG] com.xmlcalabash.extensions.Unzip - Running {http://www.daisy.org/ns/pipeline/xproc}unzip !1.17.3
2015-02-26 11:27:08,836 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:47:63:!1.17.4 read 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!EPUB/images/324123-155-007.jpg' from [pipe #9261] ([document-sequence #401 (1 docs)])
2015-02-26 11:27:08,836 [DEBUG] com.xmlcalabash.library.AddAttribute - Running add-attribute !1.17.4
2015-02-26 11:27:08,836 [TRACE] com.xmlcalabash.io.Pipe - bundle://70.0:1/xml/xproc/unzip-fileset.xpl:47:63:!1.17.4 read 'zip:file:/tmp/jobContext8150996068475231379/324123-1.epub!EPUB/images/324123-155-007.jpg' from [pipe #681] ([document-sequence #401 (1 docs)])
2015-02-26 11:27:10,359 [DEBUG] o.a.h.i.c.PoolingClientConnectionManager - Connection manager is shutting down
2015-02-26 11:27:10,360 [DEBUG] o.a.h.i.c.PoolingClientConnectionManager - Connection manager shut down
```

**Comment by josteinaj** [comment]
The issue seems to be that `px:fileset-unzip` and `px:unzip` loads all the images into memory and pipes them through the conversion.

[I may need to implement a custom unzip step](https://lists.w3.org/Archives/Public/xproc-dev/2015Feb/0010.html) (or extend the existing one) to allow unzipping directly to a directory instead of to memory.

**Comment by josteinaj** [comment]
A fix for this issue will be available when daisy/pipeline-modules-common#84 is merged and released. Until then, I will keep a copy of that fix in the nordic migrator so that we have a fix right away.

**Comment by josteinaj** [comment]
fixed in b189de1309be1446b2600ceaf9828dc6d4e3df67
